db/validation-rules.csv

```diff
@@ -0,0 +1,22 @@
+Implemented,Target,Label,Rule,Notes
```

Comment:
Do you plan to use this file programmatically?

Reply:
No I don't - csv was just a nicely manageable way to write these rules, as I didn't have a handy Markdown table editor at hand, and nowadays GitHub renders csv files quite nicely as well :smile: But this could be a Markdown file as well.

Reply:
Though, now that you mentioned it, I could have a table of labels and rule descriptions in the database, in case the user needs to check the explanation for a particular error label. But this is not relevant at least as long as I'm the only one really operating this thing. :smile: